FurEver/Pages/DogQuiz.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿@page
+@model FurEver.Pages.DogQuizModel
+@{
+    ViewData["Title"] = "Dog Breed Quiz";
+}
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <title>@ViewData["Title"]</title>
+    <link rel="stylesheet" href="~/css/site.css" />
+    <link rel="stylesheet" href="~/css/quiz.css" />
+</head>
+<body>
```

Comment:
Remove these tags.  They are already present in _Layout.cshtml, which will include this page before rendering.